tests/handler/get.test.ts

```diff
@@ -1,35 +1,36 @@
-import type {
-  APIGatewayProxyEvent, APIGatewayProxyResult, APIGatewayEventRequestContext,
-} from 'aws-lambda';
-import { v4 } from 'uuid';
-import { handler } from '../../src/handler/get';
+/* eslint-disable import/first */
+const mockSend = jest.fn();
+const mockGetParameterCommand = jest.fn();
+
+const mockSsmSend = jest.fn(() => ({
+  send: mockSend,
+}));
 
-jest.mock('../../src/util/logger.ts');
+jest.mock('@aws-sdk/client-ssm', () => ({
+  SSMClient: mockSsmSend,
+  GetParameterCommand: mockGetParameterCommand,
+}));
 
-describe('Test Get Lambda Function', () => {
-  test('should return 200 with the query parameters', async () => {
-    const queryStringParameters: Record<string, string> = { message: 'Hello world!' };
-    const requestContext: APIGatewayEventRequestContext = <APIGatewayEventRequestContext> { requestId: v4() };
-    const headers: Record<string, string> = {};
-    const eventMock: APIGatewayProxyEvent = <APIGatewayProxyEvent> {
-      queryStringParameters,
-      requestContext,
-      headers,
-    };
+import { handler } from '../../src/handler/get';
 
-    const res: APIGatewayProxyResult = await handler(eventMock);
+const headers = {
```

Comment:
Is it worth removing this and exporting the `headers` object from the get handler?